upstairs/src/downstairs.rs

```diff
@@ -964,21 +982,38 @@ impl Downstairs {
          * about that
          */
         if let Some(reconcile_list) = self.mismatch_list() {
+            self.reconcile = Some(ReconcileData {
+                id: Uuid::new_v4(),
+                reconcile_task_list_index: 0,
+            });
+
+            #[cfg(feature = "omicron-build")]
+            {
+                // If building for production, then notify Nexus when any
+                // reconciliation starts.
+                let cloned_reconcile = self.reconcile.as_ref().unwrap().clone();
```

Comment:
Nit: I think we can derive `Copy` on `ReconcileData`, which would make this a little cleaner.

Reply:
:+1: addressed in 3c0ca0d